src/app/game/interfaces/igame-round-bot.ts

```diff
@@ -0,0 +1,6 @@
+export interface IGameRoundBot {
+  name: string;
+  points: number;
+  time: number;
+  used: string;
+}
```

Comment:
There is more interfaces inside this sources than logic xD

Reply:
This is just the beginning, soon there will be more logic using these interfaces